### DISCUSSION TITLE
ADD clock to fullscreen player

### DISCUSSION THREAD

**Discussion by allanlaal** [code-for-charity/Wish-Factory#11] [Proposals & Voting, Ranking]
SHORT Table | (Summary)     
------------ | -------------   
*Problem*     |   no hardware for accurate datetime in human brain                               
*Solution*     |   show ´hh:mm´ clock in fullscreen     
*Alternatives*|    
*Scope*         |           
*Side effects*|        
*Context*      | using yt as a TV, mostly fullscreen

**Comment by allanlaal** [comment]
code: https://github.com/oxodao/YoutubeClock/blob/master/YoutubeClock.user.js

**Comment by allanlaal** [comment]
imho the update interval can be longer than 500ms, one does not need sciencey precision at 3AM, merely the fact that its 3AM is surprise enough

**Comment by mrangjunboon2017s** [comment]
why?

**Comment by allanlaal** [comment]
@mrangjunboon2017s why show a clock in fullscreen?

**Comment by allanlaal** [comment]
@ImprovedTube I was thinking more of a simple `HH:MM:SS` 24h clock and for Youtube specifically (since its the only thing thats fullscreen (on my "TV" PC))

**Comment by allanlaal** [comment]
@ImprovedTube yes, HH:MM

**Comment by sJKy79** [comment]
Chrome has a similar extension: https://chromewebstore.google.com/detail/overlay-clock/oocmeehjidegkkilebmenjdogocfefbd
